src/metrics/chaoss.ts

```diff
@@ -679,10 +701,13 @@ export const chaossNewContributors = async (config: QueryConfig<NewContributorsO
       HAVING first_time >= toDate('${config.startYear}-${config.startMonth}-1') AND first_time < toDate('${endDate.getFullYear()}-${endDate.getMonth() + 1}-1')
     )
     GROUP BY id, time
+    ${config.limitOption === 'each' && config.limit > 0 ? 
+    `${config.order ? `ORDER BY new_contributor ${config.order}` : ''} LIMIT ${config.limit} BY time` :
+    ''}
   )
   GROUP BY id
-  ${config.order ? `ORDER BY total_new_contributors ${config.order}` : ''}
-  ${config.limit > 0 ? `LIMIT ${config.limit}` : ''}
+  ${config.order ? `ORDER BY new_contributors[-1] ${config.order}` : ''}
```

Comment:
Why this `ORDER BY` Clause use `new_contributors[-1]` instead of `SUM(new_contributors)`? Is it just to be consistent with other metrics?

Reply:
Yes, the logic should be identical for all metrics. The reason using `SUM(new_contriubtors)` is that we have no identical strategy about how to sort and limit the result.

Reply:
@xgdyp Is there any more problem? I think we can also add a `orderOption` later, but for now, identical logic is fine.